README.md

```diff
@@ -1,19 +1,83 @@
+
+## Getting help and providing feedback
+
+May want to duplicate the content I put in the Elastic doc: https://github.com/elastic/enterprise-search-pubs/pull/2631
+
+## Building a connector
+
+This section has its own structure, so mini TOC to provide an overview:
+
+- [Implementing the connector protocol](#implementing-the-connector-protocol)
+- [Using the connector framework](#using-the-connector-framework)
+- [Operating the connector service](#operating-the-connector-service)
+- [Operating the connector](#operating-the-connector)
+- [Testing the integration](#testing-the-integration)
+
+### Implementing the connector protocol
+
+[//]: # (I see now that the protocol is document-based and building a connector is language- and tool-agnostic. I'm therefore leading with a section that presents a procedure to properly implement the protocol. This is the core task for the developer.)
+
+The [connector protocol](docs/CONNECTOR_PROTOCOL.md) is document-based and building a connector is language- and tool-agnostic. The procedure to properly implement the protocol without using the connector framework is as follows:
+
+- Create a new index for the connector via the Kibana UI - Enterprise Search - Create an Elasticsearch index. Use the `Build a connector` option for an ingestion method.
+- Create an API key to work with the connector. It should be done either using the `Generate API key` button, which is available on the next step of the wizard, or by creating a new API key via the Security - API keys - `Create API key`. The second way is more generic and will allow the same API key to be used for multiple connectors.
+- By this time, the documents index for the connector is already created and registered in the [.elastic-connectors](docs/CONNECTOR_PROTOCOL.md#elastic-connectors) index. It has settings, but no mappings. So if any specific mappings are needed, they should be created manually. 
+- Implement the code to synchronize the connector documents via pushing the documents directly to the index that you have created. Make sure that the data is in the format that the mappings expect.
+- Implement reading the sync schedule from the [.elastic-connectors](docs/CONNECTOR_PROTOCOL.md#elastic-connectors) and synchronize with the data source on schedule.
+- Implement the code to log sync jobs to [.elastic-connectors-sync-jobs](docs/CONNECTOR_PROTOCOL.md#jobs-index).
+- Implement updating the connector status in the [.elastic-connectors](docs/CONNECTOR_PROTOCOL.md#elastic-connectors) index at regular intervals.
+
+### Using the connector framework
```

Comment:
I'd provide more context here as well. The framework is a ruby application (the connector service), ruby libraries, and code examples that you can use to implement a connector in Ruby. To use it, you'll need a Ruby development environment.